doc_templates/phylum_auth_set-token.md

```diff
@@ -1,9 +1,4 @@
----
-title: phylum auth set-token
-category: 6255e67693d5200013b1fa3e
-parentDoc: 625758b12a1e9d0186416451
-hidden: false
----
+{PH-HEADER}
```

Comment:
If this header is removed everywhere, do we even still need it as an option?

Reply:
It think it is still useful, for a few reasons:

* It contains the `{PH-TITLE}` placeholder, which is a way to get the page title set correctly
  * The other way is to use the first header of the doc (`#` and maybe also `##`) when the `title` frontmatter is _not_ set
  * That can be tricky when the first header either doesn't exist or is meant to be something _other_ than the page title
* It can be used for future frontmatter entries, which [Docusaurus has many](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#markdown-front-matter)

Reply:
Fair enough I wasn't sure if docasaurus supported headers in similar fashion. Otherwise just using `{PH-TITLE}` should have been sufficient.